src/pages/index.tsx

```diff
@@ -1,29 +1,8 @@
 import { Box } from '@chakra-ui/react'
 
-import { withSessionSsr } from '~/lib/withSession'
-
 // TODO: Will be landing page in the future, now just a redirect to appropriate page.
 const Index = () => {
   return <Box />
```

Comment:
this is a regression (and will cause a blank page). Can revert this? We can remove `getServerSideProps` properly in another PR

Reply:
fixed by calling useUser with redirect in index

Reply:
This fix will still render a blank page if user is not signed in. We may need to redirect to `/sign-in` if user is not logged in.

Reply:
It appropriately redirects to `sign-in` in my local dev. Could you try to reprod the behaviour?

Any usage of suspense queries will automatically be caught by our top level error boundary, which should prompt the redirect to the `sign-in` page.

Reply:
Argh, that error boundary will cause a lot of ??? when other devs cannot understand why something happens since it is not obvious in this code block. Maybe add a comment here?

Reply:
tbf i think error boundaries should be used for **unexpected** errors, and we should not count on it for expected ones

Reply:
Should we still put some handling here, or at least somewhere in this code?

Reply:
We could add a comment. What would the alternative be for handling redirects? Event listeners? That's what we did on Care360

Reply:
For now comment ba. If not I was thinking of using a non-suspense version of the hook and explicitly handling errors. Not sure how errors are handled in a suspense world; maybe an error boundary really is appropriate